packages/remix-dev/compiler/js/plugins/routes_unstable.ts

```diff
@@ -130,28 +132,42 @@ export function browserRouteModulesPlugin(
 
           let sourceCode = fs.readFileSync(routeFile, "utf8");
```

Comment:
@dmarkow - What do you think of this idea? Maybe it's faster? Check for filechange by using `fs.statSync` -- before reading the file's contents -- and if stats.mtime or stats.size differ from cache, then file has changed? Could save the time of reading the whole file every time to see if it has changed.

Reply:
Surprisingly underwhelming result. Thanks for checking it out!